### PR TITLE
Merge develop to master

### PR DESCRIPTION
**Pull request by DDavidPrime**

### CODE REVIEW

**Comment by StanevPrime** [comment]
@DDavidPrime please organise a demo session before we merge this PR. Last but not least the solution needs to be tested from someone from the team to ensure that meets the needed quality.

File: packages/rx_bloc_cli/mason_templates/bricks/feature_cicd_fastlane/__brick__/docs/continuous_delivery.md [hunk 1]
```diff
@@ -82,7 +83,40 @@ Before you manually distribute artifacts for respective platforms, make sure to
 
 In order to trigger a new build, push a new tag to the repository in one of the following formats:
 `production-v1.2.3+45` or `my_awesome_tag_name-development-v1.2.3+45`
+{{/cicd_github}}
+{{#cicd_codemagic}}
+### Codemagic pipeline
+
+If the project was generated with the `--cicd=codemagic` flag, a `codemagic.yaml` file is created within the root of the project.
```
Comment (StanevPrime) [inline packages/rx_bloc_cli/mason_templates/bricks/feature_cicd_fastlane/__brick__/docs/continuous_delivery.md:90]: Please list all available workflows and their purpose

File: packages/rx_bloc_cli/mason_templates/bricks/feature_cicd_fastlane/__brick__/fastlane/Fastfile [hunk 2]
```diff
@@ -425,10 +426,10 @@ platform :ios do
             }
 
             gym(
-                workspace: PROJECT_DIRECTORY_PATH+'/ios/Runner.xcworkspace',
+                workspace: PROJECT_PATH_ABS+'/ios/Runner.xcworkspace',
```
Comment (StanevPrime) [inline packages/rx_bloc_cli/mason_templates/bricks/feature_cicd_fastlane/__brick__/fastlane/Fastfile:429]: Please confirm this works with 

1. Local distribution
2. Github Actions
3. CodeMagic

**Review by StanevPrime — CHANGES_REQUESTED** [review]

File: packages/rx_bloc_cli/example/docs/continuous_delivery.md
Comment (JvnSlv) [inline packages/rx_bloc_cli/example/docs/continuous_delivery.md:1]: Line 18.
`Before committing any changes, make sure you encrypt all the files using the `./crypto.sh encode all` command.`
The file is called `crypt.sh` either rename the file or change the documentation

**Comment by JvnSlv** [comment]
Running `rx_bloc_cli create_distribution .` overwrites the `README.md` created by the `rx_bloc_cli create` command which I think shouldn't be the case.
We should have something like:
README.md 
README_DISTRIBUTION.md

**Comment by DDavidPrime** [comment]
> Running `rx_bloc_cli create_distribution .` overwrites the `README.md` created by the `rx_bloc_cli create` command which I think shouldn't be the case. We should have something like: README.md README_DISTRIBUTION.md

How and where does this happen? The `rx_bloc_cli create_distribution .` and `rx_bloc_cli create .` should not be generated within the same directory. Since they are distinct commands, the intention is to create a separate repo for the project and the credentials, thus being two separate directories. This way there should never be a collision between the readme files.

Was that what you meant or was it some other scenario you encountered where the files were getting replaced.

**Review by StanevPrime — APPROVED** [review]

**Pull request merged**